### DISCUSSION TITLE
Handle the metroui table size

### DISCUSSION THREAD

**Discussion by AZ77-Git** [olton/Metro-UI-CSS#1807] [Q&A]
Hello,

I have created a metroui table you can see at this codepen link : https://codepen.io/AZ77/pen/RwVMVRP

I am encountring a little issue about the size : indeed, when i put this class on the div containing the table class="w-50", the page doesn't display all the content. And when i put this class on the table like this : "< table class="w-50">".... i have a strange result.

perhapse we have to set a scrollbar ?

Can you help me to have the expected behaviour ? 

I have a second little question : have you an idea about how to center the column name like the tbody please ?

**Comment by olton** [comment]
use attribute `data-hotizontal-scroll='true'`

**Reply by AZ77-Git** [reply]
Thank you. in order to help the others, I would like to notice that your are talking about data-ho**r**izontal-scroll and not data-ho**t**izontal-scroll

That work fine for me thank you. And what about the center position for the name of columns defined in the tbody ?

**Comment by AZ77-Git** [comment]
Hello,
Sorry, Can you help me about the center position of the column names ? 

Thank you

**Reply by olton** [reply]
Use attribute `data-cls-head-cell` to define class, which will be applied to the head cell.

**Reply by olton** [reply]
https://metroui.org.ua/table.html#_table_options
